metadata-ingestion/src/datahub/ingestion/source/unity/source.py

```diff
@@ -268,10 +279,11 @@ def process_table(
 
         ownership = self._create_table_ownership_aspect(table)
 
+        lineage: Optional[UpstreamLineageClass] = None
         if self.config.include_column_lineage:
             self.unity_catalog_api_proxy.get_column_lineage(table)
             lineage = self._generate_column_lineage_aspect(dataset_urn, table)
-        else:
+        elif self.config.include_table_lineage:
```

Comment:
This seemed like a bug -- we only want to ingest lineage if this is set

Reply:
yup looks good

Reply:
nice catch